.github/workflows/test.yml

```diff
@@ -45,6 +45,9 @@ jobs:
       matrix:
         os: [ macOS-13, macOS-14 ]
         sim: [ tvOS, watchOS ]
+        include:
+          - os: macos-14
+            sim: visionOS
```

Comment:
If you add here:
```
            xcode_version: 15.2
```
then before you run the shell script you can add a step:
```
      - if: ${{ matrix.xcode_version != '' }}
        run: sudo xcode-select -s /Applications/Xcode_${{ matrix.xcode_version }}.app
```
This is what I've done in my CI.